pkg/azure/client.go

```diff
@@ -165,11 +166,26 @@ func (s *Store) GetSASURL(ctx context.Context, containerPath string, containerTy
 		s.logger.Errorf("failed to marshal request body %v", err)
 		return "", err
 	}
-	req, err := http.NewRequestWithContext(ctx, http.MethodPost, fmt.Sprintf("%s/%s", global.NeuronHost, "internal/sas-token"), bytes.NewBuffer(reqBody))
+
+	u, err := url.Parse(fmt.Sprintf("%s/%s", global.NeuronHost, "internal/sas-token"))
+	if err != nil {
+		s.logger.Errorf("error while parsing endpoint %s, %v", fmt.Sprintf("%s/%s", global.NeuronHost, "internal/sas-token"), err)
```

Comment:
`fmt.Sprintf("%s/%s", global.NeuronHost, "internal/sas-token"` store this in a variable and avoid duplicacy